src/tools/compiletest/src/header.rs

```diff
@@ -860,7 +861,7 @@ pub fn make_test_description<R: Read>(
         ignore |= !has_tsan && config.parse_name_directive(ln, "needs-sanitizer-thread");
         ignore |= !has_hwasan && config.parse_name_directive(ln, "needs-sanitizer-hwaddress");
         ignore |= !has_memtag && config.parse_name_directive(ln, "needs-sanitizer-memtag");
-        ignore |= config.target_panic == PanicStrategy::Abort
+        ignore |= (config.target_panic == PanicStrategy::Abort || !has_unwind_support)
```

Comment:
Why is the `config.target_panic` check insufficient here? My intuition is that the targets in question should output `panic="abort"` for `rustc --print=cfg`.

Reply:
`config.target_panic` apparently doesn't get set based off of `rustc --print=cfg`, only base off of command-line flags.

Reply:
Got it. We should still be trying to use `--print=cfg` output here, but I guess it works for now.